src/js/components/Ballot/OfficeItemCompressed.jsx

```diff
@@ -390,13 +391,15 @@ class OfficeItemCompressed extends Component {
         <a className="anchor-under-header" name={weVoteId} />
         <div className="card-main__content">
           {/* Desktop */}
-          <Title onClick={this.gotoOfficeLink}>
-            {ballotItemDisplayName}
-            <ArrowForwardIcon
-              className="u-show-desktop"
-              classes={{ root: classes.cardHeaderIconRoot }}
-            />
-          </Title>
+          <Link to={this.getOfficeLink()}>
+            <Title onClick={this.gotoOfficeLink}>
```

Comment:
I'm wondering if the onClick on this Title is necessary given it is wrapped with a Link?

Reply:
Nice catch, I meant to remove it because it wasn't firing.